packages/hooks/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
```

Comment:
I updated CHANGELOG myself. We are still not set on the proper recommendation here, but there is new emerging pattern to list new changes as coming from `master`. Probably something good to document. /cc @aduth